rust/src/dcerpc/dcerpc.rs

```diff
@@ -38,54 +38,12 @@ pub const DCERPC_UUID_ENTRY_FLAG_FF: u16 = 0x0001;
 pub const PFC_FIRST_FRAG: u8 = 0x01;
 // Value to indicate last fragment
 pub const PFC_LAST_FRAG: u8 = 0x02;
-// Cancel was pending at sender
```

Comment:
I don't really see defines like this as dead code, would rather have them stay. It will make things easier when support is added.

Reply:
Could we use a rather Rusty convention and prefix with with `_` so its clear to the reader they are not used? And perhaps awaiting use?

Reply:
That would work for me